packages/datadog-plugin-cucumber/test/index.spec.js

```diff
@@ -41,15 +41,15 @@ const runCucumber = (version, Cucumber, requireName, featureName, testName) => {
   return cli.run()
 }
 
-describe('Plugin', () => {
+describe('Plugin', function () {
   let Cucumber
-
-  withVersions(plugin, '@cucumber/cucumber', version => {
+  this.timeout(10000)
```

Comment:
Why was this not needed before? Is there a performance regression with the new approach?

Reply:
that's a good question. Could it be because of the `ritmReset: false`? That seems to be needed with the new plugin system.

Reply:
that's a good question. Could it be because of the `ritmReset: false`? That seems to be needed with the new plugin system.

Reply:
that's a good question. Could it be because of the `ritmReset: false`? That seems to be needed with the new plugin system.

Reply:
Possible, I'm not sure the impact of that new option. If you're confident that's it I won't block on it.